controller/tests/controller_server/csi_controller_server_test.py

```diff
@@ -16,92 +16,110 @@
         context = utils.FakeContext()
 
         self.request.secrets = {"password": "pass", "management_address": "mg"}
-        self.get_create_object_method()(self.request, context)
+        self.get_tested_method()(self.request, context)
         self.assertEqual(context.code, grpc.StatusCode.INVALID_ARGUMENT, "username is missing in secrets")
         self.assertTrue("secret" in context.details)
 
         self.request.secrets = {"username": "user", "management_address": "mg"}
-        self.get_create_object_method()(self.request, context)
+        self.get_tested_method()(self.request, context)
         self.assertEqual(context.code, grpc.StatusCode.INVALID_ARGUMENT, "password is missing in secrets")
         self.assertTrue("secret" in context.details)
 
         self.request.secrets = {"username": "user", "password": "pass"}
-        self.get_create_object_method()(self.request, context)
+        self.get_tested_method()(self.request, context)
         self.assertEqual(context.code, grpc.StatusCode.INVALID_ARGUMENT, "mgmt address is missing in secrets")
         self.assertTrue("secret" in context.details)
 
         self.request.secrets = []
 
-    def _test_create_object_with_array_connection_exception(self, storage_agent):
+    def _test_request_with_array_connection_exception(self, storage_agent):
         storage_agent.side_effect = [Exception("error")]
         context = utils.FakeContext()
-        self.get_create_object_method()(self.request, context)
+        self.get_tested_method()(self.request, context)
         self.assertEqual(context.code, grpc.StatusCode.INTERNAL, "connection error occured in array_connection")
         self.assertTrue("error" in context.details)
 
-    def _test_create_object_with_get_array_type_exception(self, storage_agent):
+    def _test_request_with_get_array_type_exception(self, storage_agent):
         storage_agent.return_value = self.storage_agent
         context = utils.FakeContext()
         self.detect_array_type.side_effect = [array_errors.FailedToFindStorageSystemType("endpoint")]
-        self.get_create_object_method()(self.request, context)
+        self.get_tested_method()(self.request, context)
         self.assertEqual(context.code, grpc.StatusCode.INTERNAL, "failed to find storage system")
         msg = array_errors.FailedToFindStorageSystemType("endpoint").message
         self.assertTrue(msg in context.details)
 
+    def _test_request_with_wrong_parameters(self, storage_agent):
+        storage_agent.return_value = self.storage_agent
+        context = utils.FakeContext()
+        self.request.parameters = {"pool": "pool1"}
+        self.get_tested_method()(self.request, context)
+        self.assertNotEqual(context.code, grpc.StatusCode.INVALID_ARGUMENT)
+
+        self.request.parameters = {"capabilities": ""}
+        self.get_tested_method()(self.request, context)
+        self.assertEqual(context.code, grpc.StatusCode.INVALID_ARGUMENT, "capacity is missing in secrets")

```

Comment:
what is the meaning of this string?

Reply:
I didnt actually wrote the test, just moved it from create_volume to the `Common` tests

Reply:
I see, but you are using it, and I think it would be better to fix a common function rather than writing and using a similar but valid function

Reply:
done

Reply:
I [sill see](https://github.com/IBM/ibm-block-csi-driver/blob/7adaecee45fe926ab778450721836d46a0d7c10e/controller/tests/controller_server/csi_controller_server_test.py#L104) the issue - is it something valid? what is your input about this?

Reply:
I dont know, someone put it there, I guess its for log purposes

Reply:
it looks wrong to me, please remove it, at least where you use it

Reply:
done